About.html

```diff
@@ -0,0 +1,165 @@
+            alt="menu-icons"
+            id="menu-icon"
+          />
+        </div>
+        <div class="main-menu-container">
+          <div class="main-menu" id="main-menu">
+            <span class="chef-icon"
+              ><a href="index.html"
+                ><img
+                  src="./images/phone-view-icons/download.png"
+                  alt="chef-icon" /></a
+            ></span>
+            <span class="close"
+              ><img
+                src="./images/phone-view-icons/x.svg"
+                alt="close-icon"
+                id="close-icon"
+            /></span>
+            <ul class="menu-list">
+              <li><a href="index.html" id="about">Home</a></li>
+              <li><a href="#">Program</a></li>
+              <li><a href="#">Join</a></li>
+              <li><a href="#">Sponsor</a></li>
+              <li><a href="#" class="button" id="button">CC Campaign</a></li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <section class="Welcome-container">
+      <div class="about-container">
+        <h2 class="about-title">THIERRY'S RESTAURANT 2.0</h2>
+        <p class="about-para1">
+          I am very happy to present you my Restaurant. it's a place with a
+          master chef graduated from the high college of cooked in London. I
+          will really love to have all of you by my side that the of the
+          opening...
+        </p>
+        <p class="about-para2">
+          Do you have any orther question about this event please you can
+          contact us through! <br />
+          <a href="#" class="email">noumbuht@gmail.com</a>
+        </p>
+      </div>
+    </section>
+    <section class="about-container-logo">
+      <h3 class="logo-text">THIERRY'S RESTAURANT 2.0 Logo</h3>
+      <div class="orange-line"></div>
+      <p class="logo-para">
+        Through the logo contest, which was held from July 5th to July 7th, the
+        winning design was determined through online voting and evaluation by
+        judges. The logo of THIERRY'S RESTAURANT 2.0, which was optimized for
+        use on the homepage and promotional materials after consultation with
+        the winning designer, is as follows.
+      </p>
+      <img
+        src="./images/phone-view-icons/download.png"
+        alt="chef-icon"
+        class="logo-img"
+      />
+    </section>
+    <section class="about-recipt-container">
+      <h3 class="logo-text">View our Menu</h3>
+      <div class="orange-line"></div>
+      <p class="logo-para">
+        Among the menu offered on the table this is just one of the nice food we
+        can showcase here.
+      </p>
+      <div class="desktop-recipt">
+        <div class="recipt1">
+          <img src="./images/phone-view-icons/achu.jpeg" alt="achu-img" />
+        </div>
+        <div class="recipt2">
+          <img src="./images/phone-view-icons/ndoe.jpg" alt="ndole-img" />
+        </div>
+      </div>
+    </section>
```

Comment:
- Kindly, Increase the size of the menu image, make it bluer, and arrange the space between the view menu and partner section to stick to the given design and have a better UI 👍 
- Please consider the updating the mobile version also 👍 
#### your design 👇
![Screenshot 2023-07-06 141702](https://github.com/axcell-0/first-capstone/assets/52010789/ab08bb0b-6f74-4a41-b7de-148901dd5029)

#### GIven Design 👇
![Screenshot 2023-07-06 141532](https://github.com/axcell-0/first-capstone/assets/52010789/50f34aca-0ae4-4756-8982-67b032fbcc98)